extensions/wikia/Recirculation/RecirculationController.class.php

```diff
@@ -20,6 +21,7 @@ public function index() {
 		$posts = $fandomDataService->getPosts( $this->type );
 
 		if ( count( $posts ) > 0 ) {
+			$this->response->setCacheValidity( self::CACHE_DURATION );
```

Comment:
for 100 req/min:
15 min cache - 99,99333333% requests hits varnish
30 min cache - 99,99666667% requests hits varnish

Does this change is really worth adding another standard time?